internal/security/secretstore/password_test.go

```diff
@@ -81,9 +81,9 @@ func TestRetrieveCred(t *testing.T) {
 
 	configuration := &config.ConfigurationStruct{
 		SecretService: secretstoreclient.SecretServiceInfo{
-			Server: parsed.Hostname(),
-			Port:   port,
-			Scheme: "https",
+			Server:   parsed.Hostname(),
+			Port:     port,
+			Protocol: "https",
```

Comment:
Shouldn't we use "http" here?

Reply:
Negative, this is for the already existing test case.